docs/install-kafka-operator.md

```diff
@@ -35,74 +35,102 @@ This method uses a command-line tool of the commercial [Banzai Cloud Supertubes]
+    ```
 
-Verify that the cert-manager pods have been created:
+### Install zookeeper-operator with Helm {#install-zookeeper-operator-with-helm}
 
-```bash
-kubectl get pods -n cert-manager
-```
+{{< kafka-operator >}} requires [Zookeeper](https://zookeeper.apache.org) for Kafka. Deploy a Zookeeper cluster if you don't already have one.
 
-Expected output:
+> Note: You are recommended to create a separate Zookeeper deployment for each Kafka cluster. If you want to share the same Zookeeper cluster across multiple Kafka cluster instances, use a unique zk path in the KafkaCluster CR to avoid conflicts (even with previous defunct KafkaCluster instances).
 
-```bash
-NAME                                      READY   STATUS    RESTARTS   AGE
-cert-manager-7747db9d88-vgggn             1/1     Running   0          29m
-cert-manager-cainjector-87c85c6ff-q945h   1/1     Running   1          29m
-cert-manager-webhook-64dc9fff44-2p6tx     1/1     Running   0          29m
-```
+1. If you are installing {{< kafka-operator >}} on Red Hat OpenShift, you need to elevate the permissions of the namespace of the Zookeeper service account.
 
-### Install Zookeeper {#install-zookeeper}
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccounts:zookeeper
+    ```
 
-Kafka requires [Zookeeper](https://zookeeper.apache.org). Deploy a Zookeeper cluster if you don't already have one.
+    In case you are using a non-default namespace for Zookeeper, you need to modify the command to take that namespace.
 
-> Note: You are recommended to create a separate Zookeeper deployment for each Kafka cluster. If you want to share the same Zookeeper cluster across multiple Kafka cluster instances, use a unique zk path in the KafkaCluster CR to avoid conflicts (even with previous defunct KafkaCluster instances).
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccounts:{NAMESPACE_FOR_ZOOKEEPER_SERVICE_ACCOUNT}
+    ```
 
 1. Install Zookeeper using the [Pravega's Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
 
     ```bash
-    helm repo add pravega https://charts.pravega.io
-    helm repo update
-    helm install zookeeper-operator --namespace=zookeeper --create-namespace pravega/zookeeper-operator
+    helm install \
+    zookeeper-operator \
+    --repo https://charts.pravega.io zookeeper-operator \
+    --version 0.2.14 \
+    --namespace=zookeeper \
+    --create-namespace \
+    --atomic \
+    --debug
     ```
 
+1. Verify that zookeeper-operator has been deployed and is in running state.
+
+    ```bash
+    kubectl get pods --namespace zookeeper
+    ```
+
+    ```bash
+    NAME                                  READY   STATUS    RESTARTS   AGE
+    zookeeper-operator-5857967dcc-gm5l5   1/1     Running   0          3m22s
+    ```
+
+### Deploy a Zookeeper cluster for Kafka {#deploy-a-zookeeper-cluster-for-kafka}
+
 1. Create a Zookeeper cluster.
 
     {{< include-code "create-zookeeper.sample" "bash" >}}
```

Comment:
I think we should add the reclaimPolicy: Delete configuration for the create-zookeeper.sample file.
Because currently what you wrote: (even with previous defunct KafkaCluster instances) will not good because of the persistent PVs

Reply:
Fixed in https://github.com/banzaicloud/koperator-docs/pull/89/commits/d02a96e200682566abf57c65b5f6ed44c79bc56f.